ledger/acctupdates_test.go

```diff
@@ -1352,6 +1352,191 @@ func TestBoxNamesByAppIDs(t *testing.T) {
 	}
 }
 
+func TestBoxCache(t *testing.T) {
+	partitiontest.PartitionTest(t)
+
```

Comment:
@algoidurovic Minor and optional:  Is there a by-design reason _not_ to run the test in parallel?

Running `gotestsum -- ./ledger` locally several times works without issue.  Cursory inspection suggests to me the test does not rely on global state changes.

```suggestion
	partitiontest.PartitionTest(t)
        t.Parallel()

```